spec/unit/azurerm_server_show_spec.rb

```diff
@@ -39,10 +58,50 @@
                 "Offer", @server.properties.storage_profile.image_reference.offer, 
                 "Sku", @server.properties.storage_profile.image_reference.sku, 
                 "Version", @server.properties.storage_profile.image_reference.version, 
-                "OS Type", @server.properties.storage_profile.os_disk.os_type]
+                "OS Type", @server.properties.storage_profile.os_disk.os_type,
+                "Public IP address", @public_ip_data.properties.ip_address,
+                "FQDN", @public_ip_data.properties.dns_settings.fqdn]
 
     expect(@arm_server_instance.service.ui).to receive(:list).with(details, :columns_across, 2)
     @arm_server_instance.run  
   end 
 
-end
\ No newline at end of file
+  it "should give display Server Name, Size, Provisioning State, Location, Publisher, Offer, Sku, Version, OS Type. Public IP address & FQDN value is empty as Public Ip address is not allocated to VM" do
```

Comment:
It would be better to say: `it "should display empty Public IP address and FQDN when Public IP address is not allocated to the VM"`